google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/SinkInfo.java

```diff
@@ -580,13 +580,7 @@ public static SinkInfo of(String name, Destination destination) {
 
   LogSink toPb(String projectId) {
     LogSink.Builder builder =
-        LogSink.newBuilder()
-            .setName(name)
-            .setDestination(destination.toPb(projectId))
-            .setOutputVersionFormat(
-                versionFormat == null
-                    ? LogSink.VersionFormat.VERSION_FORMAT_UNSPECIFIED
-                    : versionFormat.toPb());
+        LogSink.newBuilder().setName(name).setDestination(destination.toPb(projectId));
```

Comment:
Don't you think it should be something like this:
```java
LogSink.newBuilder().setName(name).setDestination(destination.toPb(projectId)).setOutputVersionFormat(versionFormat.toPb())
```
OR
```java
LogSink.newBuilder().setName(name).setDestination(destination.toPb(projectId)).setOutputVersionFormat(LogSink.VersionFormat.V2)
```

To support the scenario when `versionFormat` is `null`

Reply:
in both the cases setOutputVersionFormat method is deprecated we have to remove it, that's the issue.

Reply:
My bad, I forgot the deprecated method itself.